include/deal.II/boost_adaptors/bounding_box.h

```diff
@@ -22,8 +22,10 @@
 
 #include <deal.II/boost_adaptors/point.h>
 
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
 #include <boost/geometry.hpp>
```

Comment:
I thought you wanted to get rid of that header inclusion?

Reply:
Separate PR, though.

Reply:
Good point. I was mostly concerned that point.h (which is included everywhere across the library) pulls it in, though.